.vscode/settings.json

```diff
@@ -1,10 +1,10 @@
 // Place your settings in this file to overwrite default and user settings.
 {
   "editor.codeActionsOnSave": {
+    "source.prettier.forceFormatDocument": true,
```

Comment:
Is this a new recommended best practice from the VS Code team or just preference?

Reply:
No, it's not. I'm going to undo this difference if it is not necessary.

This difference has two intent.
First, The intent of this difference is to resolve #1277 by realizing the following quoted portion.

> Originally posted by @rohit-gohri in https://github.com/prettier/prettier-vscode/issues/1277#issuecomment-621175180 is a working solution for running prettier before eslint.
> 
> The above solution runs the default formatter (which can be set to Prettier) then uses the code action for eslint afterwards.
> https://github.com/prettier/prettier-vscode/issues/1555#issue-702228085

Second, it is for testing the created new code action in this PR.
The intention was to make it clear that I was testing through the code action.

Reply:
It looks like Microsoft's recommended settings

> Although you can also use the formatter on save using the setting editor.formatOnSave it is recommended to use the editor.codeActionsOnSave feature since it allows for better configurability.

https://github.com/microsoft/vscode-eslint